src/main/kotlin/fi/metatavu/timebank/api/forecast/ForecastService.kt

```diff
@@ -80,4 +80,31 @@ class ForecastService {
     fun getHolidays(): String? {
         return doRequest("/v1/holiday_calendar_entries")
     }
+
+    /**
+     * Gets projects from Forecast
+     *
+     * @return Response with projects data
+     */
+    fun getProjects(): String? {
+        return doRequest("/v1/projects")
+    }
+
+    /**
+     * Gets tasks from Forecast
+     *
+     * @param pageNumber page of paginated response to request
+     * @return Response with tasks data
+     */
+    fun getTasks(pageNumber: Int): String? {
+        return doRequest("/v4/tasks?pageSize=1000&pageNumber=$pageNumber")
+    }
+
+    companion object {
+        const val NON_BILLABLE = "NON_BILLABLE"
+        const val FIXED_PRICE = "FIXED_PRICE"
+        const val FIXED_PRICE_V2 = "FIXED_PRICE_V2"
```

Comment:
FIXED_PRICE_V2?

Reply:
These values are budget types that Forecast API returns. There seems to be some projects using FIXED_PRICE_V2.

Reply:
Although project budget types turned out to be redundant since all the information we need can be get from Tasks rather than Projects. This whole companion object should've been removed and will be in next commit to this PR.